Gemfile

```diff
@@ -87,7 +87,7 @@ platforms :ruby do
   group :db do
     gem 'pg', '>= 0.18.0'
     gem 'mysql', '>= 2.9.0'
-    gem 'mysql2', '>= 0.3.13'
+    gem 'mysql2', '>= 0.3.18'
```

Comment:
0.3.18 includes fix msec for time field is not passed. https://github.com/brianmario/mysql2/pull/579